src/components/file-input/file-input.stories.tsx

```diff
@@ -0,0 +1,320 @@
+/* eslint-disable @typescript-eslint/no-use-before-define */
+import React, { useState, useRef } from "react";
+import { ComponentStory } from "@storybook/react";
+
+import FileInput, { FileUploadStatusProps } from ".";
+
+export const Default: ComponentStory<typeof FileInput> = () => {
+  return <FileInput label="File input" />;
+};
+
+export const Margins: ComponentStory<typeof FileInput> = () => {
```

Comment:
Fair comment. I added it because basically the first several stories I added were "copies" of those in Textbox that still seemed relevant for FileInput. I wasn't even sure whether to have the Margin Props, or whether I should add Padding Props too - I expected quite a discussion on that tbh 😁 I'll hold my hands up and say I just went for consistency with Textbox (which also has this story), without thinking about it too much - that could be the wrong approach as they're obviously pretty different as components!

Having made my excuses, I agree this story isn't really adding much for consumers so happy to remove it 👍